homework-02/my-calculator-v01/src/cli.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+const calculator = require('my-calculator-v01');
```

Comment:
I would recommend to require `./index`, otherwise it's kind of a circular dependency